pilot/pkg/config/kube/crd/conversion.go

```diff
@@ -45,6 +46,22 @@ func FromJSON(s collection.Schema, js string) (config.Spec, error) {
 	return c, nil
 }
 
+func IstioStatusJSONFromMap(jsonMap map[string]interface{}) (config.Status, error) {
```

Comment:
use `StatusJSONFromMap` instead of `IstioStatusJSONFromMap`?

Reply:
Its pretty specific to `IstioStatus`, and there are other types of statuses.